src/inverseRobot.js

```diff
@@ -24,7 +24,27 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  for (const mainKey in robot) {
+    let counter = 0;
+
+    for (const key in robot) {
+      if (robot[mainKey] === robot[key]) {
+        counter++;
+      }
+    }
+
+    if (counter > 1) {
+      return null;
+    }
+  }
+
+  const inrsRobot = {};
```

Comment:
'inrsRobot' it's difficult to understand what does it mean, 'inversedRobot' not much longer but definitely clearly)